developer-docs-site/docs/tools/aptos-cli-tool/aptos-db-restore.md

```diff
@@ -0,0 +1,66 @@
+# Aptos DB Restore Tools and Public Backup Files
+
+Since its launch in October 2022, the Aptos community has grown rapidly. As of May 2023, Aptos has 743G and 159G of data in testnet and mainnet, respectively. We expect the data to increase greatly as more transactions are submitted to the blockchain. Facing the large amount of data, we want to provide users with a way to achieve two goals:
+
+- Quickly bootstrap a database to start a new or failed node
+- Efficiently recover data from any specific period
+
+Our DB restore tool enables you to use existing public backup files to restore the database on your local machine. These public backup files, which have cryptographic proof, are stored on both AWS and Google Cloud for public download. You can use these backup files, along with our restore tool, to restore your database to any historical range or to the latest version.
+
+## **Restore DB using the Public Backup Files**
+
+Our CLI supports restoring a database using backup files. It reads from the backup files and recreates the Aptos DB. We support two kinds of restore: (1) recreating a DB with minimal transaction history at a user-specified transaction version (or the latest version the backup has), and (2) restoring the database over a specific period. In addition to (1), this option also ensures that the recreated DB carries the ledger history of the user-designated version range.
+
+**Bootstrap DB**
+
+The command restores the database from the closest snapshot to the target version. This command can quickly restore a database to a target version, but it does not restore all the transaction history from the past.
+
+Here is an example command (note: depending on whether you use AWS or Google Cloud, you may need to follow the instructions to install [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) or [gsutil](https://cloud.google.com/storage/docs/gsutil_install) as prerequisites):
+
+```bash
+# This requires follow prerequsites
+# 1. aws cli or google gsutil installed
+# 2. aptos cli 1.0.14	
+aptos node bootstrap-db \ 
+    --target-version 500000000 \
```

Comment:
Can we elaborate on what this will produce? i.e., will it sync the last `500000000` txns? Or will it sync from genesis to `500000000`?